src/renderer/transaction/transactions-history/index.jsx

```diff
@@ -28,94 +30,201 @@ import {
-		return () => {
-			this.updateCopyText(itemIndex, this.copiedText);
-			const bounceTime = setTimeout(() => {
-				this.updateCopyText(itemIndex, this.copyText);
-				clearTimeout(bounceTime);
-			}, 1000);
-		};
+	renderDate(timestamp) {
+		const { year, month, day } = getAbrDateFromTimestamp(timestamp);
+		return (
+			<div>
+				{day} {month} {year}
+			</div>
+		);
 	}
 
 	render() {
-		const { transactions } = this.props;
-		const { page, currentCopyValues } = this.state;
+		const { transactions, classes } = this.props;
+		const { rowsPerPage, page } = this.state;
+
 		return (
 			<Paper>
-				<Toolbar>
+				<Toolbar className={classes.toolbar}>
 					<Typography variant="h6">Transactions</Typography>
-					<IconButton aria-label="Refresh">
+					<IconButton aria-label="Refresh" onClick={this.handleRefresh}>
 						<RefreshIcon />
 					</IconButton>
 				</Toolbar>
 				<Table>
 					<TableBody>
-						{transactions.slice(page).map(transaction => {
+						{paginate(transactions, rowsPerPage, page).map(transaction => {
 							return (
 								<TableRow key={transaction.id}>
 									<TableCell>
-										{this.renderIcon(transaction.statusIconName)}
+										{getIconForTransaction(
+											transaction.statusIconName,
+											transaction.sending
+										)}
 									</TableCell>
-									<TableCell>{transaction.date}</TableCell>
-									<TableCell>{transaction.statusText}</TableCell>
-									<TableCell align="right">
-										{transaction.cryptoCurrency}
+									<TableCell>
+										<Typography
+											component="span"
+											variant="body2"
+											color="secondary"
+											gutterBottom
+										>
+											{this.renderDate(transaction.timeStamp)}
+										</Typography>
+									</TableCell>
+									<TableCell>
+										<Typography component="span" variant="body2" gutterBottom>
+											{transaction.statusText ||
+												getCustomStatusText(transaction)}
+										</Typography>
 									</TableCell>
 									<TableCell align="right">
-										<CopyToClipboard
-											text={transaction.externalLink}
-											onCopy={this.handleOnCopy(transaction.id)}
-										>
-											<div>
-												<CopyIcon />
-												<span>
-													{' '}
-													{currentCopyValues[transaction.id] ||
-														this.copyText}{' '}
-												</span>
-											</div>
-										</CopyToClipboard>
+										<Typography component="span" variant="body2" gutterBottom>
+											{transaction.sending ? '- ' : '+ '}
+											{transaction.value
+												? transaction.value.toLocaleString()
+												: ''}
+										</Typography>
 									</TableCell>
 									<TableCell align="right">
-										<Button size="small" disableRipple>
-											<ViewIcon />
-											View
-										</Button>
+										<IconButton className={classes.rightSpace}>
+											<CopyToClipboard
+												text={`https://ropsten.etherscan.io/tx/${
```

Comment:
@eduardosmoura this should be `https://etherscan.io/tx/` for production and `ropsten` for dev mode. You can check which is currently being used by the wallet at run time through `config.chainId` from `common`. Cheers